### PR TITLE
Add disk space recommendations for pulp migration

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: plugins/katello/3.18/upgrade/index.md [hunk 1]
```diff
@@ -78,6 +78,16 @@ If the above steps failed, please review /var/log/foreman-installer/katello.log
 
 Katello 3.18 supports migrations of Content to Pulp 3 for container, yum, and file content.  Depending on how much content is present, this can take a very long time.  The largest part of the process can be run without downtime.  This full process will need to be complete before upgrading to Katello 4.0.
 
+### File system requirements
+
+Upgrading to pulp3 does require some additional space, although the bulk of the content is not duplicated.  Ensure you have:
+
+1.  80% of the space /var/lib/mongodb/ consumes should be available in /var/opt/rh/rh-postgresql12/lib/pgsql/.  (If /var/lib/mongodb is 100GB, then 80GB needs to be free for postgresql).
```
Comment (ianballou) [inline plugins/katello/3.18/upgrade/index.md]: I'd reword point 1 to sound more like point 2.  Point 1 sounds a little funny following "Ensure that you have ..."

File: plugins/katello/nightly/upgrade/index.md [hunk 2]
```diff
@@ -78,6 +78,15 @@ If the above steps failed, please review /var/log/foreman-installer/katello.log
 
 Katello 3.18 supports migrations of Content to Pulp 3 for container, yum, and file content.  Depending on how much content is present, this can take a very long time.  The largest part of the process can be run without downtime.  This full process will need to be complete before upgrading to Katello 4.0.
 
+### File system requirements
+
+Upgrading to pulp3 does require some additional space, although the bulk of the content is not duplicated.  Ensure you have:
+
+1.  80% of the space /var/lib/mongodb/ consumes should be available in /var/opt/rh/rh-postgresql12/lib/pgsql/.  (If /var/lib/mongodb is 100GB, then you need 80GB free for postgresql).
```
Comment (ianballou) [inline plugins/katello/nightly/upgrade/index.md]: (Ditto)

File: plugins/katello/3.18/upgrade/index.md [hunk 3]
```diff
@@ -78,6 +78,16 @@ If the above steps failed, please review /var/log/foreman-installer/katello.log
 
 Katello 3.18 supports migrations of Content to Pulp 3 for container, yum, and file content.  Depending on how much content is present, this can take a very long time.  The largest part of the process can be run without downtime.  This full process will need to be complete before upgrading to Katello 4.0.
 
+### File system requirements
+
+Upgrading to pulp3 does require some additional space, although the bulk of the content is not duplicated.  Ensure you have:
```
Comment (ianballou) [inline plugins/katello/3.18/upgrade/index.md]: Swap "pulp3" for "Pulp 3" ?

**Comment by jlsherrill** [comment]
thanks @ianballou .  Updated!

File: plugins/katello/3.18/upgrade/index.md [hunk 4]
```diff
@@ -78,6 +78,16 @@ If the above steps failed, please review /var/log/foreman-installer/katello.log
 
 Katello 3.18 supports migrations of Content to Pulp 3 for container, yum, and file content.  Depending on how much content is present, this can take a very long time.  The largest part of the process can be run without downtime.  This full process will need to be complete before upgrading to Katello 4.0.
 
+### File system requirements
+
+Upgrading to Pulp 3 does require some additional space, although the bulk of the content is not duplicated.  Ensure you have:
+
+1.  80% of the space /var/lib/mongodb/ consumes available in /var/opt/rh/rh-postgresql12/lib/pgsql/.  (If /var/lib/mongodb is 100GB, then 80GB needs to be free for postgresql).
+2.  200% of the space /var/lib/pulp/published/ available within /var/lib/pulp/ (If /var/lib/pulp/published is taking 10GB, 20GB needs to be free in /var/lib/pulp/)
```
Comment (ianballou) [inline plugins/katello/3.18/upgrade/index.md]: Just needs "consumes" now

Comment (jlsherrill) [reply plugins/katello/3.18/upgrade/index.md]: done!

**Review by ianballou — APPROVED** [review]
Looks good to me!
